google-cloud-video-stitcher/samples/acceptance/media_cdn_key_definition.rb

```diff
@@ -0,0 +1,12 @@
+require "google/cloud/video/stitcher"
+
+def media_cdn_def cdn_key_path, hostname, key_name, private_key
```

Comment:
Please put this in a helper module rather than adding it to the Object class. It might be easiest just to add it to the base class in `helper.rb` rather than putting it in a separate file like this. (I know the other definitions are doing this Object-method-in-a-separate-file thing, but they shouldn't be. Can we at least establish a better precedent here with this new definition?)

Reply:
Moved all of the definitions to `helper.rb`